components/AvalancheProblemImage.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import {Text, Image, StyleSheet, TouchableHighlight, useWindowDimensions, View, ActivityIndicator} from 'react-native';
+import {Text, Image, StyleSheet, /*TouchableHighlight,*/ useWindowDimensions, View, ActivityIndicator} from 'react-native';
```

Comment:
Feel free to just delete the thing - `git revert` is easy and won't leave comments in here.